docs/samples/v1beta1/torchserve/v2/bert/README.md

```diff
@@ -8,12 +8,14 @@ on KServe.
 Clone [pytorch/serve](https://github.com/pytorch/serve) repository,
 navigate to `examples/Huggingface_Transformers` and follow the steps for creating the MAR file including serialized model and other dependent files.
 TorchServe supports both eager model and torchscript and here we save as the pretrained model. 
+
+Download the preprocess script from [here](./sequence_classification/Transformer_handler_generalized_v2.py)
  
 ```bash
 torch-model-archiver --model-name BERTSeqClassification --version 1.0 \
 --serialized-file Transformer_model/pytorch_model.bin \
---handler ./Transformer_handler_generalized.py \
---extra-files "Transformer_model/config.json,./setup_config.json,./Seq_classification_artifacts/index_to_name.json"
+--handler ./Transformer_handler_generalized_v2.py \
+--extra-files "Transformer_model/config.json,./setup_config.json,./Seq_classification_artifacts/index_to_name.json,./Transformer_handler_generalized.py"
```

Comment:
Do we need `Transformer_handler_generalized.py` in the extra files?

Reply:
Yes , `Transformer_handler_generalized.py` is needed in extra files. `Transformer_handler_generalized_v2.py` inherits from `Transformer_handler_generalized.py`. 

Can you please try with `s3://kubeflow-dataset/kserve-0.5.3/bert` and let me know if you face any errors ?

Reply:
@yuzisun can you pls check if your transformers version is 4.11.0. Also, pls check if your setup_config is similar to [this one](https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/setup_config.json).

Reply:
Yes. The [torchserve example](https://github.com/pytorch/serve/tree/master/examples/Huggingface_Transformers) works only with `transformers==4.6.0`. 

And please set the `captum_explanations` to `false` in `setup_config.json` until we get the captum part ready.

Reply:
Added an instruction in readme file as well

Reply:
tested the infer endpoint with your mar file it works. I was using the model from the original example bucket not sure if that was the issue.

Reply:
> > @shrinath-suresh can you help squash the commits?
> 
> sure Dan . I will do it after addressing your comments.

Thanks Dan. i have squashed the commits.